cdci_data_analysis/analysis/parameters.py

```diff
@@ -61,6 +61,14 @@ def subclasses_recursive(cls):
         indirect.extend(subclasses_recursive(subclass))
     return direct + indirect
 
+def basic_check_bounds(val, min_value = None, max_value = None, name=None):
```

Comment:
perhaps I would rename this as `basic_numeric_check_bounds`, as it the implementation is the most suitable for numeric values

Reply:
There is no implementation for non-numerics now. But ok, it could be added for e.g. time

Reply:
Indeed it's just a naming adjustment